jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/card/CardImageTest.java

```diff
@@ -0,0 +1,76 @@
+package io.github.epam.bootstrap.tests.composite.section.card;
+
+import io.github.epam.TestsInit;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static io.github.com.StaticSite.bsPage;
+import static io.github.com.pages.BootstrapPage.cardExample;
+import static io.github.com.pages.BootstrapPage.cardImage;
+import static io.github.epam.bootstrap.tests.BaseValidations.baseValidation;
+import static io.github.epam.states.States.shouldBeLoggedIn;
+import static org.hamcrest.Matchers.is;
+import static org.testng.Assert.assertEquals;
+
+/**
+ * Created by Natalia Amelina on 27.09.2019
+ * Email: natalia_amelina@epam.com; Skype: nat_amelina
+ */
+public class CardImageTest extends TestsInit {
+
+    private static final String SRC_ATTR_EXPECTED = "https://jdi-testing.github.io/jdi-light/images/wolverin.jpg";
+    private static final String ALT_ATTR_EXPECTED = "image";
+    private static final String IMAGE_TOP_CLASS = "card-img-top";
+    private static final String TEXT = "Some quick example text.";
+    private static final int WIDTH = 86;
+    private static final int HEIGHT = 137;
+
+    @BeforeMethod
+    public void before() {
+        shouldBeLoggedIn();
+        bsPage.shouldBeOpened();
+    }
+
+    @Test
+    public void getMainTextTest() {
+        assertEquals(cardImage.text.getText(), TEXT);
+    }
+
+    @Test
+    public void availabilityest() {
+        cardImage.text.is().enabled().displayed();
```

Comment:
to have opportunity to set up debug point we should methods potential to fail put on the new lines, enabled() and displayed() are potential methods to fail, so it should looks like:
```
cardImage.text.is()
     .enabled()
     .displayed();
```
so now if we want, we can set up debug point on each line with such methods